tests/test-mmt.ipynb

```diff
@@ -0,0 +1,71 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# WMTS Tests"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from maap_test_helper import *\n",
+    "import requests"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# papermill parameters cell\n",
+    "\n",
+    "stage = \"dit\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not stage in [ \"dit\", \"main\", \"production\" ]:  # SKIP until dit and production are fixed\n",
```

Comment:
MMT should not be down in DIT or production?

DIT: https://mmt.dit.maap-project.org/
Production: https://mmt.maap-project.org/

Reply:
Fixed